study.md

```diff
@@ -62,21 +69,37 @@ delicious apple.
 Copy the JSON that you wrote for the previous question and include the new information about the kitchen.
 
 ```json
-<!-- your answer here -->
-```
+{
+  'kitchen': {
+      'cupboard': {'bowls', 'plates'}
+      'stove': {'small', 'medium', 'large', 'xl'}
+      'spiceRack':
+        'rackOne':{'chives', 'lemon grass', 'mint'},
+        'rackTwo': {'cinnamon', 'nutmeg', 'vanilla'}
+      'fruitBasket': {
+        'fujiApples':{
+          'quantity': {2}
```

Comment:
careful here, 'quantity' is pointing to an object but the object doesn't have any key value pairs, it only has the number 2.